benchmark/benchmark_pytext_vocab.py

```diff
@@ -0,0 +1,199 @@
+import os
+import sys
+from collections import (Counter, OrderedDict)
+import time
+from typing import List, Union
+
+# this is needed because we want to add 'torchtext/examples/data_pipeline' directory to the
+# `sys.path` variable in order to import the pytext_vocab (since its not a module)
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "examples", "data_pipeline"))
+
+from pytext_vocab import ScriptVocab as ExperimentalScriptVocabulary
+from pytext.torchscript.vocab import ScriptVocabulary as PytextScriptVocabulary
```

Comment:
Do we add the dependency of the pytext library.

Reply:
I didn't do this because I saw that your pipeline benchmarking PR with the `ScriptVocabulary` from pytext didn't add the dependency either. Do you think I should? Especially because this benchmark would just be for an example class that we aren't maintaining anyways?

Reply:
We probably won't add pytext dependency because we don't use any work from pytext in our main library.